Developers/Executing_Tests.rst

```diff
@@ -28,7 +28,7 @@ coala project directory.
 
 ::
```

Comment:
hey, firstly, you ignored my 2nd point, it should be Removes not Remove.

secondly, i didnt mean to type literal "url_link", just Closes and then the link